rqt_gui_cpp/src/rqt_gui_cpp/nodelet_plugin_provider.cpp

```diff
@@ -34,114 +34,99 @@
 
 #include "roscpp_plugin_provider.h"
 
-#include <ros/callback_queue.h>
-#include <ros/ros.h>
-
 #include <stdexcept>
-#include <boost/bind.hpp>
 
 namespace rqt_gui_cpp {
 
 NodeletPluginProvider::NodeletPluginProvider(const QString& export_tag, const QString& base_class_type)
```

Comment:
Should there be a separate PR to rename these classes?

Reply:
Yes